R/update_pbp_participation.R

```diff
@@ -73,11 +73,14 @@ pbp_participation <-
         number_of_pass_rushers,
         players_on_play
       ) |>
+      # oak -> lv exception
+      dplyr::mutate(tmp_possession_team = dplyr::case_when(possession_team == "OAK" ~ "LV",
```

Comment:
Maybe more long term robust to use nflreadr:clean_team_abbrs()?

Reply:
I agree here. 

`nflreadr::clean_team_abbrs(possession_team, current_location = TRUE, keep_non_matches = TRUE)`

would be more robust in case of future team abbreviation changes. 

The current solution looks fine imo. It's just less robust

Reply:
Much better, wasn't familiar with that particular function. Thanks for the tip!